packages/docusaurus-theme-translations/locales/tk/theme-search-algolia.json

```diff
@@ -0,0 +1,35 @@
+{
+  "theme.SearchBar.label": "Gözleg",
+  "theme.SearchBar.seeAll": "Ähli netijeleri gör ({count})",
+  "theme.SearchModal.errorScreen.helpText": "Internet birikmäňizi barlaň.",
+  "theme.SearchModal.errorScreen.titleText": "Gözleg netijelerini ýüklemek mümkin däl",
+  "theme.SearchModal.footer.closeKeyAriaLabel": "Escape düwmesi",
+  "theme.SearchModal.footer.closeText": "ýapmak",
+  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Aşak ok düwmesi",
+  "theme.SearchModal.footer.navigateText": "nawigasiýa",
+  "theme.SearchModal.footer.navigateUpKeyAriaLabel": "Ýokary ok düwmesi",
+  "theme.SearchModal.footer.searchByText": "Gözleg",
+  "theme.SearchModal.footer.selectKeyAriaLabel": "Enter düwmesi",
+  "theme.SearchModal.footer.selectText": "saýlamak",
+  "theme.SearchModal.noResultsScreen.noResultsText": "Sorag boýunça netijeler ýok",
+  "theme.SearchModal.noResultsScreen.reportMissingResultsLinkText": "Bize habar beriň.",
+  "theme.SearchModal.noResultsScreen.reportMissingResultsText": "Gözleg netijesi ýokmy?",
+  "theme.SearchModal.noResultsScreen.suggestedQueryText": "Synanyşyň",
+  "theme.SearchModal.placeholder": "Gözleg",
+  "theme.SearchModal.searchBox.cancelButtonText": "Ýatyrmak",
+  "theme.SearchModal.searchBox.resetButtonTitle": "Arassalamak",
+  "theme.SearchModal.startScreen.favoriteSearchesTitle": "Halaýanlar",
+  "theme.SearchModal.startScreen.noRecentSearchesText": "Gözleg taryhy ýok",
+  "theme.SearchModal.startScreen.recentSearchesTitle": "Soňky",
+  "theme.SearchModal.startScreen.removeFavoriteSearchButtonTitle": "Halaýan ýazgyny aýyrmak",
+  "theme.SearchModal.startScreen.removeRecentSearchButtonTitle": "Taryh ýazgysyny aýyrmak",
+  "theme.SearchModal.startScreen.saveRecentSearchButtonTitle": "Gözleg soragyny ýatda saklaň",
+  "theme.SearchPage.algoliaLabel": "Algolia tarapyndan gözleg",
+  "theme.SearchPage.documentsFound.plurals": "{count} dokument|{count} dokumentler|{count} dokumentler",
```

Comment:
2 forms instead of 3?

Reply:
There are 2, but Turkmen has six cases: nominative, (specifically definite) accusative, dative, genitive, instrumental/ablative and locative. And all the plurals are formed from -lar/ler, but it is not used when the quantity is explicit, but also plurals can also change according to these six cases. There will be one exceptional situation for accusative case that changes plural form, which makes it up to 3 forms in total. 
I know why they didn't provide this information on the website link you provided, the reason for this is that in there they provided information only for the countable numbers and not the suffixes of the noun which is followed by that certain number.
And you can also mention that in the translations I provided not in all the accusative forms it changes the form, sometimes it stays the same (if compared to English, plurals in Turkmen are like 3 forms of past tenses in English, for some it changes, but for some it does not. For example: (cut, cut, cut) - does not change, (worked, worked, worked) - does not change, (write, wrote, written) - changes). That is the reason why I made it 3 forms. Because for some nouns and cases it changes the form and for some it does not. 
Should I redo that according to only those situations that exactly changes the plural? Because currently they have their third form regardless if it changes or not.